.huskyrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "lint-staged"
```

Comment:
Das ist die Konfigurationsdatei von Husky, eines der Tools, das ich oben schon angesprochen habe.

Husky erlaubt es bestimmte Kommandos mit git hooks zu koppeln. In diesem Fall führen wir nur was beim commit aus, theoretisch kann man das aber bei jedem anderen Befehl auch machen (z.B. bei checkout, merge, push usw.). Wenn wir das in Zukunft wollen, könnte man das hier machen. Ein weiterer Anwendungsfall wäre z.B. vor jedem Push die Tests laufen zu lassen, oder so.

Hier führen wir nur `lint-staged` aus, das erkläre ich in der nächsten Datei.